x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/type/EsqlDataTypes.java

```diff
@@ -44,6 +44,7 @@ public final class EsqlDataTypes {
 
     public static final DataType DATE_PERIOD = new DataType("DATE_PERIOD", null, 3 * Integer.BYTES, false, false, false);
     public static final DataType TIME_DURATION = new DataType("TIME_DURATION", null, Integer.BYTES + Long.BYTES, false, false, false);
+    public static final DataType GEO_POINT = new DataType("geo_point", Double.BYTES * 2, false, false, false);
```

Comment:
I don't understand why the size of the field here is 2 doubles and we treat it everywhere as a long, what is the relationship? how is this information used?

Reply:
Yes, I spotted that a while back and planned to fix it. I'll look again and see if I can get clarity on this.